cadquery/occ_impl/exporters/svg.py

```diff
@@ -128,7 +129,7 @@ def getSVG(shape, opts=None):
         Export a shape to SVG
     """
 
-    d = {"width": 800, "height": 240, "marginLeft": 200, "marginTop": 20}
```

Comment:
This is just a minor style thing, but if we intend for a user to modify these options, I'd suggest just making them keyword args to the function. CQ already uses much more arguments in https://github.com/CadQuery/cadquery/blob/53a76d4f288c4babd416dfb3b2c3e7b9153f754f/cadquery/occ_impl/shapes.py#L1761-L1805, I don't see any reason to handle them a different way here.

Reply:
I was trying to be consistent with what has been done previously with the `opt` parameter. However, I can structure it according to the consensus. I don't have any real attachment to the current `opt` method.

Reply:
I think I also prefer flat options. Maybe a deprecation warning and temporary support for both would be a good consensus?

Reply:
Adding keyword args ended up being a dead-end for now, as @adam-urbanczyk expected. I would have to rework too much of the `export.exporters` interface and I wasn't willing to do that at this time. I tried to put as much detail in the `getSVG()` docstring as possible to offset this at least a little bit.